app/grandchallenge/components/validate_scripts/validate_biom.py

```diff
@@ -0,0 +1,45 @@
+import sys
+from pathlib import Path
+
+
+class ValidationError(Exception):
+    pass
+
+
+def run(biom_file_path):
+    # Import non-buildins locally
+    import biom
+    import h5py
```

Comment:
Why is this necessary?

Reply:
Originally: to prevent a red-herring of 'could not import' or vague HDF5-library clashes and rather show a runtime error that indicates one is trying to import from a script.

I forgot we had a forbidden import construct, I think that is superior and I will follow that route.